### PR TITLE
don't allow users to put URLs in transcription text

### PR DESCRIPTION
**Pull request by rstorey**
Refs #213

### CODE REVIEW

File: concordia/views.py [hunk 1]
```diff
@@ -50,6 +51,7 @@
 ASSETS_PER_PAGE = 36
 PROJECTS_PER_PAGE = 36
 ITEMS_PER_PAGE = 36
+URL_REGEX = "http[s]?://"
```
Comment (acdha) [inline concordia/views.py]: I try to always prefix regular expressions with `r` to mark them as [raw string literals](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals). In addition to not needing to escape back-slashes, in many editors and on GitHub that activates regex syntax highlighting:

```suggestion
URL_REGEX = r"http[s]?://"
```

File: concordia/views.py [hunk 2]
```diff
@@ -398,6 +400,19 @@ def save_transcription(request, *, asset_pk):
     else:
         user = request.user
 
+    # Check whether this transcription text contains any URLs
+    # If so, ask the user to correct the transcription by removing the URLs
+    transcription_text = request.POST["text"]
+    urls = re.search(URL_REGEX, transcription_text)
```
Comment (acdha) [inline concordia/views.py]:
```suggestion
    url_match = re.search(URL_REGEX, transcription_text)
```

File: concordia/views.py [hunk 3]
```diff
@@ -398,6 +400,19 @@ def save_transcription(request, *, asset_pk):
...
+    if urls:
```
Comment (acdha) [inline concordia/views.py]:
```suggestion
    if url_match:
```

File: concordia/views.py (hunk 1 above)
Comment (rstorey) [reply concordia/views.py]: I meant to do that, but I forgot

**Pull request merged**